float64 target_rpm